packages/feedback/src/integration.ts

```diff
@@ -185,6 +188,29 @@ export class Feedback implements Integration {
     }
   }
 
+  /**
+   * Hook that is called after all integrations are setup. This is used to
+   * integrate with the Replay integration to save a replay when Feedback modal
+   * is opened.
```

Comment:
Hmm is that correct? Doesn't this just trigger on init, when the integration is initialized? Which is already done by replay itself anyhow?

Reply:
Not in the case where sampling rates are both 0

Reply:
but that seems weird to me, if I add feedback it will overwrite it if I manually set the sample rates to 0? e.g.:

```js
Sentry.init({
  replaysErrorSampleRate: 0,
  replaysSessionSampleRate: 0,
  integrations: [Sentry.replaysIntegration(), Sentry.feedbackIntegration()]
})
```

? I don't think that makes sense. We should be buffering in 99% of cases anyhow, and if users _really_ don't want it to buffer, they probably have a reason for it...?

Reply:
If they really don't want to buffer, they can remove the replay integration. I suppose if they could be doing some manual buffering cases, but then can then turn it off with `includeReplay: false` to the feedback integration.

Reply:
We've decided to split the sampling change out and settle how to handle that separately from this PR.